docs/whats_new/v1.0.0.md

```diff
@@ -0,0 +1,305 @@
+it is also more stable thanks to our [new usage of Ray subprocesses](https://github.com/autogluon/autogluon/pull/3614) during low-memory training,
+resulting in **0 task failures** on the AutoML Benchmark.
+
+AutoGluon 1.0 is capable of achieving the fastest inference throughput of any AutoML system while still obtaining state-of-the-art results.
+By specifying the `infer_limit` fit argument, users can trade off between accuracy and inference speed to meet their needs.
+
+As seen in the below plot, AutoGluon 1.0 sets the Pareto Frontier for quality and inference throughput, achieving Pareto Dominance compared to all other AutoML systems.
+AutoGluon 1.0 High achieves superior performance to AutoGluon 0.8 Best with 8x faster inference and 8x less disk usage!
+
+![AutoGluon 1.0 AutoML Benchmark Plot](https://raw.githubusercontent.com/Innixma/autogluon-benchmark/master/v1_results/pareto_front_all_v1_w_v08.png)
+
+We would like to conclude this spotlight by thanking Pieter Gijsbers, Sébastien Poirier, Erin LeDell, Joaquin Vanschoren,
+and the rest of the AutoML Benchmark authors for their key role in 
+providing a shared and extensive benchmark to monitor the progress of the AutoML field.
+Their support has been invaluable to the AutoGluon project's continued growth.
+
+We would also like to thank Frank Hutter, who continues to be a leader within the AutoML field, for organizing
+the AutoML Conference in 2022 and 2023 to bring the community together to share ideas and align on a compelling vision.
+
+We are excited to see what our users can accomplish with AutoGluon 1.0's enhanced performance.
+As always, we will continue to refine and enhance AutoGluon in future releases to push the boundaries of AutoML forward for all.
+
+## Spotlight: Newly Published Papers
+
+### AutoGluon-TimeSeries: AutoML for Probabilistic Time Series Forecasting
+
+We have published a [paper on AutoGluon-TimeSeries](https://openreview.net/forum?id=XHIY3cQ8Tew) at AutoML Conference 2023 ([YouTube Video](https://www.youtube.com/watch?v=niLmfjXeHnE)).
+In the paper, we benchmarked AutoGluon and popular open-source forecasting frameworks (including DeepAR, TFT, AutoARIMA, AutoETS, AutoPyTorch).
+AutoGluon produces SOTA results in point and probabilistic forecasting, and even **achieves 65% win rate against the best-in-hindsight combination of models**.
+
+### TabRepo: A Large Scale Repository of Tabular Model Evaluations and its AutoML Applications
+
+We have published a paper on Tabular Zeroshot-HPO ensembling simulation to arXiv ([Paper Link](https://arxiv.org/pdf/2311.02971.pdf), [GitHub](https://github.com/autogluon/tabrepo)).
+This paper is key to achieving the performance improvements seen in AutoGluon 1.0, and we plan to continue to develop the code-base to support future enhancements.
+
+### XTab: Cross-table Pretraining for Tabular Transformers
+
+We have published a paper on tabular Transformer pre-training at ICML 2023 ([Paper Link](https://arxiv.org/abs/2305.06090)).
```

Comment:
Can we add github link here as well? Potentially this one: https://github.com/BingzhaoZhu/XTab

Reply:
added